examples/examples-gpu/nyc-taxi/rf-rapids-dask.ipynb

```diff
@@ -4,9 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>worth adding that the GPU has 16GB of GPU RAM</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
added this note in https://github.com/saturncloud/examples/pull/58/commits/9b7dd6a56f88d660d9cc3fe2d4724c4879e196a3